fuzz.go

```diff
@@ -61,6 +61,27 @@ func NewWithSeed(seed int64) *Fuzzer {
 	return f
 }
 
+// NewFromGoFuzz is a helper function that enables using gofuzz (this
+// project) with go-fuzz (https://github.com/dvyukov/go-fuzz) for continuous
+// fuzzing. Essentially, it enables translating the fuzzing bytes from
+// go-fuzz to any Go object using this library.
```

Comment:
Add a note that we promise that the mapping from a given []byte to an object will be constant (if you use the same fuzz settings & object schema, of course) over future upgrades.